learning/tour-of-beam/frontend/lib/components/footer.dart

```diff
@@ -114,6 +110,39 @@ class _PrivacyPolicyButton extends StatelessWidget {
   }
 }
 
+class _BeamVersion extends StatelessWidget {
+  const _BeamVersion();
+
+  Future<String?> _getSdkBeamVersion() async {
+    final sdk = GetIt.instance.get<AppNotifier>().sdk;
+    if (sdk == null) {
+      return null;
+    }
+    final metadata = await GetIt.instance.get<CodeClient>().getMetadata(sdk);
+    return metadata.beamSdkVersion;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AnimatedBuilder(
+      animation: GetIt.instance.get<AppNotifier>(),
+      builder: (context, child) => FutureBuilder<String?>(
+        // TODO(nausharipov) review: is this ok? I didn't want to initialize a future in initState.
+        // ignore: discarded_futures
+        future: _getSdkBeamVersion(),
+        builder: (context, snapshot) => snapshot.hasData
+            ? Text(
+                '${'ui.builtWith'.tr()} v${snapshot.data}',
```

Comment:
Add a named argument to the translated string.
`'ui.builtWith'.tr(namedArgs: {'beamVersion': snapshot.data})` or something like that.